app/javascript/packages/document-capture/components/document-capture-review-issues.tsx

```diff
@@ -37,6 +37,7 @@ function DocumentCaptureReviewIssues({
 }: DocumentCaptureReviewIssuesProps) {
   const { t } = useI18n();
   const { isSelfieCaptureEnabled } = useContext(SelfieCaptureContext);
+  const isReviewStep = true;
```

Comment:
I think adding a variable for this slightly hurts readability, can we remove it and inline `isReviewStep={true}` below?
```suggestion
```

Reply:
I tried setting it to true in the tag, but lint complained

Reply:
Looks like the complaint was about switching `isReviewStep={true}` to just `isReviewStep`. This works for me and passes `yarn run lint`

```diff
diff --git a/app/javascript/packages/document-capture/components/document-capture-review-issues.tsx b/app/javascript/packages/document-capture/components/document-capture-review-issues.tsx
index 37dc8af89..7489c732f 100644
--- a/app/javascript/packages/document-capture/components/document-capture-review-issues.tsx
+++ b/app/javascript/packages/document-capture/components/document-capture-review-issues.tsx
@@ -37,7 +37,6 @@ function DocumentCaptureReviewIssues({
 }: DocumentCaptureReviewIssuesProps) {
   const { t } = useI18n();
   const { isSelfieCaptureEnabled } = useContext(SelfieCaptureContext);
-  const isReviewStep = true;
 
   const defaultSideProps = {
     registerField,
@@ -80,16 +79,12 @@ function DocumentCaptureReviewIssues({
           ]}
         />
       )}
-      <DocumentsCaptureStep
-        defaultSideProps={defaultSideProps}
-        value={value}
-        isReviewStep={isReviewStep}
-      />
+      <DocumentsCaptureStep defaultSideProps={defaultSideProps} value={value} isReviewStep />
       {isSelfieCaptureEnabled && (
         <SelfieCaptureStep
           defaultSideProps={defaultSideProps}
           selfieValue={value.selfie}
-          isReviewStep={isReviewStep}
+          isReviewStep
         />
       )}
       <FormStepsButton.Submit />
```

Reply:
👀